content/deployment.md

```diff
@@ -50,6 +50,40 @@ Environment variables are used to set process specific things, which could conce
 
 A final note on storing these settings: It's not a good idea to store settings the same repository where you keep your app code. Read about good places to put your settings in the [Security article](security.html#api-keys).
 
+<h4 id="environment-variable-list">List of environment variables</h4>
+
+Here's a list of the environment variables you can provide to your application.
+
+`BIND_IP` 
+Bind the application server to a specific interface, for example: `BIND_IP=192.168.0.2`.
+
+`DISABLE_WEBSOCKETS` 
+Set this to 1 to disable WebSocket usage in your application.
+
+`HTTP_FORWARDED_COUNT` 
+Set this to however many network hops you have running before your Meteor application. For example, if have a nginx server acting as a proxy before your Meteor application, you would set `HTTP_FORWARDED_COUNT=1`. If you have a load balancer in front of that nginx server, the count is 2.
+
+`MAIL_URL` 
```

Comment:
Some of these, like `MAIL_URL` are already defined in other places, although I think this might be a better description!  Regardless, it might make sense to not double-up the descriptions (extra work to update in the future), so maybe some of those should link here?